overwatch/src/services/state.rs

```diff
@@ -167,9 +170,12 @@ where
     }
 }
 
-impl<State, Operator> StateHandle<State, Operator> {
-    pub fn new(initial_state: State, operator: Operator) -> (Self, StateUpdater<State>) {
-        let (sender, receiver) = channel(initial_state);
+impl<State, Operator> StateHandle<State, Operator>
+where
+    State: Clone,
+{
+    pub fn new(operator: Operator) -> (Self, StateUpdater<State>) {
+        let (sender, receiver) = channel(5);
```

Comment:
Where was this `5` taken from? Could that be a parameter or a const generic of this?

Reply:
In fact the state should only be the **latest** state, hence a single sized buffer (?)

Reply:
It's an arbitrary number.
It's a consequence of shifting the internal channel from `watch` to `broadcast`.
I think we can continue this conversation in the thread above, as it's directly related to it.